src/main/java/org/jenkinsci/plugins/github/util/BuildDataHelper.java

```diff
@@ -1,105 +0,0 @@
-package org.jenkinsci.plugins.github.util;
-
-import hudson.model.Job;
-import hudson.model.Run;
-import hudson.plugins.git.Revision;
-import hudson.plugins.git.util.Build;
-import hudson.plugins.git.util.BuildData;
-import org.eclipse.jgit.lib.ObjectId;
-
-import javax.annotation.Nonnull;
-import java.io.IOException;
-import java.util.List;
-import java.util.Set;
-
-/**
- * Stores common methods for {@link BuildData} handling.
- *
- * @author Oleg Nenashev <o.v.nenashev@gmail.com>
- * @since 1.10
- */
-public final class BuildDataHelper {
```

Comment:
What is the reason for deleting class? If you want to get new methods, add them here and Deprecate old, either this will break binary compatibility for plugins that using it.